cinema/views.py

```diff
@@ -43,6 +47,38 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(string: str) -> list[int]:
+        return [int(str_id) for str_id in string.split(",")]
+
+    def get_queryset(self):
+        queryset = self.queryset
+
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+
+        if self.action in ("list", "retrieve"):
+            queryset = queryset.prefetch_related("genres")
+            queryset = queryset.prefetch_related("actors")
```

Comment:
can u write this in one prefetch_related ?

Reply:
yes. united